recipes/herbie-data/meta.yaml

```
@@ -13,6 +13,9 @@ build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
+  script_env:
+      - export "PYTHONIOENCODING=utf8"  # [linux]
```

Just one line 
```suggestion
      - PYTHONIOENCODING=utf8
```
This key handles doing it as appropriate per-OS.

---

Ah, gotcha. I'll try that. Thanks for stepping me through this :)